chia/farmer/farmer.py

```diff
@@ -50,6 +50,7 @@
 
 UPDATE_POOL_INFO_INTERVAL: int = 3600
 UPDATE_POOL_FARMER_INFO_INTERVAL: int = 300
+UPDATE_HARVESTER_CACHE_INTERVAL: int = 60
```

Comment:
Should any of these be in config.yaml?

Reply:
They can be, but no need, since users should not see any effect from changing these variables